packages/vuetify/src/styles/utilities/_index.sass

```diff
@@ -1,6 +1,7 @@
 @import './_display.sass'
 @import './_text.sass'
 @import './_transition.sass'
+@import './_screenreaders.sass'
```

Comment:
transpose with line 3

Reply:
Do you mean line 2 to keep imports ordered alphabetically?